pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -325,12 +325,16 @@ class Build {
                         case "windows": buildWindowsInstaller(versionData); break
                 	default: return; break
                 }
-		try {
-                    context.sh 'for file in $(ls workspace/target/*.tar.gz workspace/target/*.pkg workspace/target/*.msi); do sha256sum "$file" > $file.sha256.txt ; done'
-                    writeMetadata(versionData)
-                    context.archiveArtifacts artifacts: "workspace/target/*"
-                } catch (e) {
-                    context.println("Failed to build ${buildConfig.TARGET_OS} installer ${e}")
+                // Archive the Mac and Windows pkg/msi
+                if (buildConfig.TARGET_OS == "mac" || buildConfig.TARGET_OS == "windows") {
+		    try {
+                        context.sh 'for file in $(ls workspace/target/*.tar.gz workspace/target/*.pkg workspace/target/*.msi); do sha256sum "$file" > $file.sha256.txt ; done'
+                        writeMetadata(versionData)
+                        context.archiveArtifacts artifacts: "workspace/target/*"
+                    } catch (e) {
+                        context.println("Failed to build ${buildConfig.TARGET_OS} installer ${e}")
+                        currentBuild.result = 'FAILURE'
```

Comment:
This line should probably be removed as this was what was causing us a lot of false failures this morning https://github.com/AdoptOpenJDK/openjdk-build/pull/1787

Reply:
Is that directly related to this issue Morgan? That sounds like a separate PR if it's required

Reply:
The PR is adding it back into the master